### PR TITLE
Fix .readthedocs.yaml and update README.md

### PR DESCRIPTION
**Pull request by austin-bowen**
As mentioned in #444, it seems the docs on readthedocs haven't been updated in the last couple releases. This PR tries to fix this by adding some missing fields to the `.readthedocs.yaml` file.

### CODE REVIEW

File: .readthedocs.yml [hunk 1]
```diff
@@ -1,4 +1,22 @@
...
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
 version: 2
+
+# Set the OS, Python version and other tools you might need
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3"
```
Comment (austin-bowen) [inline .readthedocs.yml:11]: I found in a recent project of mine that I needed these fields for the build to work. Python version `"3"` should just use the latest stable Python version. The rest of the additions in this file are from the [template](https://docs.readthedocs.io/en/stable/config-file/v2.html).

**Comment by pndurette** [comment]
@austin-bowen Oh thanks for this! I saw your PR on my phone when you sent it but never got back to it. Indeed ReadtheDocs changed stuff recently, but never really paid enough attention, didn't even realize the docs weren't updating, oops. LGTM!

**Comment by pndurette** [comment]
The issue w/ RTD was two fold: your fix above and also the Webhook was broken and needed to be resync'd. Sadly the tag v2.5.0 push was too far ago, so I can't redeliver that payload, but `latest` (`main`) docs have built successfully.

**Comment by austin-bowen** [comment]
Sweet! Glad we figured it out 👍

**Comment by pndurette** [comment]
Fixes #444

**Pull request merged**